scalahost/sbt/src/sbt-test/sbt-scalahost/semantic-example/test

```diff
@@ -0,0 +1,11 @@
+> app/testOnly Fat
```

Comment:
How about we say `set every scalametaSemanticdb := ScalametaSemanticdb.Fat` before this line?